api/app/signals/apps/api/v1/serializers/nested/type.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework import serializers
+
+from signals.apps.api.generics.serializers import SIAModelSerializer
+from signals.apps.signals.models import Type
+
+
+class _NestedTypeModelSerializer(SIAModelSerializer):
+    code = serializers.CharField(source='name', max_length=3)
+
+    class Meta:
+        model = Type
+        fields = (
+            'code',
```

Comment:
Also add the `created_by` in the serialzier, we do this in all other serializers so to be consistent also add it here